src/screens/homeScreen/tests.js

```diff
@@ -0,0 +1,11 @@
+const insulinDeliveryScreen = require('./insulinDeliveryScreen').tests;
+const activeCarbohydratesScreen = require('./activeCarbohydratesScreen').tests;
+const activeInsulinScreen = require('./activeInsulinScreen').tests;
+const homeScreen = require('./HomeScreen').tests;
+
+module.exports = {
+  homeScreen,
+  activeInsulinScreen,
+  activeCarbohydratesScreen,
```

Comment:
Am I correct to assume that test, for example the `activeCarbohydratesScreen` will live here at some point?

Reply:
Actually it will be the opposite. I'm planning on completely removing this file, removing all the tests from the screens and keeping all the tests in the E2E folder. I'll delete this file when I have finished moving everything over.

Reply:
actually i should just rename e2e. to tests....